autoPyTorch/pipeline/components/preprocessing/image_preprocessing/normalise/base_normalizer.py

```diff
@@ -11,9 +11,11 @@ class BaseNormalizer(autoPyTorchImagePreprocessingComponent):
 
     def __init__(self) -> None:
         super(BaseNormalizer, self).__init__()
-        self._fit_requirements = [FitRequirement('channelwise_mean', np.ndarray),
-                                  FitRequirement('channelwise_std', np.ndarray)]
-        self._fit_requirements.extend(super()._fit_requirements)
+        self._fit_requirements = [FitRequirement('channelwise_mean', (np.ndarray,)),
+                                  FitRequirement('channelwise_std', (np.ndarray,))]
+        super_requirements: Optional[List[FitRequirement]] = super().get_fit_requirements()
+        if super_requirements:
+            self._fit_requirements.extend(super_requirements)
```

Comment:
Thinking about this a bit more, it would probably be more elegant to provide an `add_requirement` method in the base component Then all components just have to call add requirement in their init. That way, one does not have to be careful to extend the super requirements when adding a new component.